### ISSUE TITLE
Supress not used test & benchmark functions

### ISSUE THREAD

**Issue by rekby** [golangci/golangci-lint#109]
_test.go files have Benchmark... and Test... functions - it isn't use in main program and linters warning many times about dead code.

I think these errors have to be supress by default for these functions.

**Comment by jirfag** [comment]
Thank you, it looks like a bug of linters, can you give more information? Maybe you can give minimal reproducible example or full file?
In all repos I’ve tested it doesn’t report about test functions

**Comment by rekby** [comment]
https://github.com/rekby/url-short-prototype-go/blob/5a6e50f1e6b754ce5c2ed64e79cec9f4e49fdbae/tarantool_bench_test.go

https://github.com/rekby/url-short-prototype-go/blob/5a6e50f1e6b754ce5c2ed64e79cec9f4e49fdbae/storage_tarantool_test.go

**Comment by rekby** [comment]
Minimal example:

bug repeated for file test_test.go

```golang
package main

import "testing"

func TestTest(t *testing.T) {

}
```

```bash
golangci-lint --version
golangci-lint has version 1.7.2 built from 93311ae on 2018-06-19T13:43:26Z

golangci-lint run
test_test.go:5:6: `TestTest` is unused (deadcode)
func TestTest(t *testing.T) {
     ^
```

**Comment by IvanBoyko** [comment]
Still not fixed?
